src/main/java/org/zalando/pazuzu/feature/FeaturesResource.java

```diff
@@ -32,10 +32,16 @@ public FeaturesResource(FeatureService featureService) {
     @RequestMapping(method = RequestMethod.GET, produces = MediaType.APPLICATION_JSON_VALUE)
     public List<FeatureDto> listFeatures(
             @RequestParam(required = false, name = "name") String[] featureNames,
-            @RequestParam(required = false, name = "sorted") Integer sorting)
+            @RequestParam(required = false, name = "sorted") Integer sorting,
```

Comment:
I agree it's not part of that PR. But the discussion how to solve '1 endpoint 2 use-cases' problem is still open.